packages/form-config/src/helpers/utils.tsx

```diff
@@ -0,0 +1,344 @@
+import {
+  URLParams,
+  OpenSRPService,
+  getFetchOptions,
+  HTTPError,
+  GetAccessTokenType,
+  processAcessToken,
+} from '@opensrp/server-service';
+import { Dictionary } from '@onaio/utils';
+import { UploadFileFieldTypes } from './types';
+import {
+  OPENSRP_FORMS_ENDPOINT,
+  OPENSRP_FORM_METADATA_ENDPOINT,
+  OPENSRP_MANIFEST_ENDPOINT,
+} from '../constants';
+import { ERROR_OCCURRED } from '../lang';
+import {
+  fetchManifestFiles,
+  ManifestFilesTypes,
+  removeManifestFiles,
+} from '../ducks/manifestFiles';
+import { handleDownload } from './fileDownload';
+import { fetchManifestDraftFiles, removeManifestDraftFiles } from '../ducks/manifestDraftFiles';
+import { Dispatch } from 'redux';
+import { fetchManifestReleases, ManifestReleasesTypes } from '../ducks/manifestReleases';
+
+type StrNum = string | number;
+
+/**
+ * format long date to YYY-mm-dd
+ *
+ * @param {string} stringDate - date as a string
+ * @returns {string} - string of date in YYY-mm-dd format
+ */
+export const formatDate = (stringDate: string): string => {
+  const date = new Date(stringDate);
+  let dd: StrNum = date.getDate();
+  let mm: StrNum = date.getMonth() + 1;
+  const yyy = date.getFullYear();
+  if (dd < 10) dd = `0${dd}`;
+  if (mm < 10) mm = `0${mm}`;
+  return `${yyy}-${mm}-${dd}`;
```

Comment:
why not use date parsing utils e.g `date-fns` for this

Reply:
@p-netm Ok...But instead of introducing a new package can we use moment?

Reply:
moment seems deprecated, no?, just trying to move away form that.